src/status_im/contexts/profile/contact/contact_request/view.cljs

```diff
@@ -0,0 +1,39 @@
+(ns status-im.contexts.profile.contact.contact-request.view
```

Comment:
Not sure if this is the best name organising this code, lmk if you have any suggestions 🙏